odometry_topic: "/odometry"
calibration_topic: "/calibrate_odom"
wheel_radius: 0.08               # [m]
wheel_dist: 0.43                 # [m]
fig_name: "odom"